app/code/Magento/Store/Model/Address/RendererInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Store\Model\Address;
+
+use Magento\Framework\DataObject;
+
+/**
+ * Interface RendererInterface
+ * @package Magento\Store\Model\Address
+ */
+interface RendererInterface
```

Comment:
Such interface should have been named something like `AddressRendererInterface` or `AddressFormatterInterface`.

Reply:
I did considered this but because the interface is in the namespace Magento\Store\Model\Address it seemed unnecessary to add Address to the name. What do you think?

Reply:
The rule of thumb is (imho) - name should be verbose enough for understanding without aliasing. We already have generic `RendererInterface` which is fine as it just renders element. But here renderer is more specific.

Another example - `Collection` lying inside `Product` folder is not a good name, `ProductCollection` is much better. But there is no need to duplicate context in method names, `$address->getId()` is just fine, `$address->getAddressId()` looks inelegantly.

Anyway, this is just a side note, I believe we don't need a new interface introduced here.